CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - console: avoid count queries for large tables (#4692)
 - console: add read replica support section to pro popup (#4118)
 - console: allow modifying default value for PK (fix #4075) (#4679)
+- console: re-enable partial support for foreign tables
```

Comment:
why is it "partial"?

Reply:
Because there never was explicit support for foreign tables in the console. They were misclassified as views, meaning several features of the console never worked with them. It was kind of an "accidental support", in a way. This PR restores this, but doesn't go any further.

Reply:
Ah I see.